pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -216,21 +216,13 @@ func (b *OFBridge) CreateGroup(id GroupIDType) Group {
 }
 
 func (b *OFBridge) createGroupWithType(id GroupIDType, groupType ofctrl.GroupType) Group {
-	ofctrlGroup, err := b.ofSwitch.NewGroup(uint32(id), groupType)
-	if err != nil { // group already exists
-		ofctrlGroup = b.ofSwitch.GetGroup(uint32(id))
-	}
+	ofctrlGroup := &ofctrl.Group{ID: uint32(id), GroupType: groupType, Switch: b.ofSwitch}
 	g := &ofGroup{bridge: b, ofctrl: ofctrlGroup}
 	return g
 }
 
-// DeleteGroup deletes a specified group in groupDb.
 func (b *OFBridge) DeleteGroup(id GroupIDType) error {
```

Comment:
I don't think we need to remove `DeleteGroup` because nothing is performed in this func. My point is we are switching implementation, changing the interface is not necessary. Also, having `CreateGroup` without `DeleteGroup` is not symmetric.

Reply:
@tnqn @hongliangl what is your thought?

Reply:
Since DeleteGroup is no longer used by Antrea, I suggest removing it.